test/specs/collections/Table/TableHeader-test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import * as common from 'test/specs/commonTests'
+
+import TableHeader from 'src/collections/Table/TableHeader'
+
+describe('TableHeader', () => {
+  common.isConformant(TableHeader)
+  common.rendersChildren(TableHeader)
+
+  it('renders as a thead by default', () => {
+    shallow(<TableHeader />)
+      .should.have.tagName('thead')
+  })
+
+  describe('fullWidth', () => {
```

Comment:
Would be nice for this to just be `common.propKeyOnlyToClassName(TableHeader, 'fullWidth'), although I don't think it would work out of the box because that helper expects the prop name to be the same as the class name.

Maybe update the common function to be:
```js
// Current
it('adds prop name to className', () => {
  shallow(createElement(Component, { ...requiredProps, [propKey]: true }))
    .should.have.className(propKey)
})

// Change to
it('adds prop name to className', () => {
  shallow(createElement(Component, { ...requiredProps, [propKey]: true }))
    .should.have.className(_.kebabCase(propKey))
})
```